src/client/eppo-client.ts

```diff
@@ -984,7 +983,15 @@ export default class EppoClient {
   }
 
   public useLRUInMemoryBanditAssignmentCache(maxSize: number) {
-    this.banditAssignmentCache = new LRUInMemoryAssignmentCache(maxSize);
+    this.banditAssignmentCache = new TLRUInMemoryAssignmentCache(maxSize);
```

Comment:
did you mean to use TLRU in here as well? the method name says LRU

Reply:
while "LRU" has meaning, I wonder if "TLRU" is too vauge and we should be expliti in our naming calling it `ExpiringLRUInMemoryAssignmentCache" or something

Reply:
Also we should allow the timeout to be optionally specified here

Reply:
I was also not conviced by the naming at first. I used `ExpiringLRUInMemoryAssignmentCache` in the beginning but then while reading about different caching algorithms. stumbled upon this.
https://en.wikipedia.org/wiki/Cache_replacement_policies#Time-aware,_least-recently_used

@aarsilv I will change the name if that doesn't convince you. What do you think?